src/zenml/models/v2/plugin/event_flavor.py

```diff
@@ -14,11 +14,27 @@
 """Models representing event source flavors.."""
 from typing import Any, Dict
 
-from zenml.models.v2.plugin.plugin_flavor import BasePluginFlavorResponse
+from zenml.models.v2.plugin.plugin_flavor import (
+    BasePluginFlavorResponse,
+    BasePluginResponseBody,
+    BasePluginResponseMetadata,
+)
 
 
-class EventFlavorResponse(BasePluginFlavorResponse):
-    """Response model for Event Flavors."""
+class EventFlavorResponseBody(BasePluginResponseBody):
+    """Response body for event flavors."""
+
+
+class EventFlavorResponseMetadata(BasePluginResponseMetadata):
+    """Response metadata for event flavors."""
 
     source_config_schema: Dict[str, Any]
     filter_config_schema: Dict[str, Any]
+
+
+class EventFlavorResponse(
```

Comment:
Heh, I failed to mention this until now and now it's spreading: this should be called `EventSourceFlavorResponse`. Same thing applies to the new body/metadata classes. Even the filename should be `event_source_flavor.py`